### DISCUSSION TITLE
Expand cart fields using include and expand

### DISCUSSION THREAD

**Discussion by tarikrital** [swellstores/community#503] [Help] [answered]
Welcome, I already got include working by getting product vendor using [this](https://github.com/orgs/swellstores/discussions/496#discussioncomment-8386492) thread.

Now I want to get the cart items with vendor in items entry.

```
const guestCart = await swell.get("/carts/{id}", {
id: X,
expand: ['items.product',],
include: {
'items.vendor': { // here I don't find the exact way to append it
url: "/products/{id}/vendor",
data: {
fields: "first_name",
},
},
},
});
```

**Comment by logeshswell** [comment]
1. How does a `url` can be "/products/{id}/vendor"?
  * Shouldn't it be "/products/{id}"?
2. Can you share the product URL to give you a definite answer?
3. Do you receive any error while running the above statement?

```
'items.vendor': { // here I don't find the exact way to append it
```
Can you replace it with `items_vendor`?

**Comment by tarikrital** [comment, marked as answer]
I used the following expand because I didn't find a way to hide sensitive data from `cart.items.vendor`.

`expand: ['items.product.vendor',],`
